lib/open_mirego/endpoint.ex

```diff
@@ -1,6 +1,9 @@
 defmodule OpenMirego.Endpoint do
   use Phoenix.Endpoint, otp_app: :open_mirego
 
+  # Always serve requests from a single canonical host
+  plug PlugCanonicalHost, canonical_host: Application.get_env(:open_mirego, OpenMirego.Endpoint)[:url][:host]
```

Comment:
Good place to show you the `Kernel#get_in/2` function! Same thing as the revolutionary `Hash#dig` in Ruby 2.3 :dash: 

`get_in(Application.get_env(:open_mirego, OpenMirego.Endpoint), [:url, :host]`